.github/workflows/typespec-requirement.yaml

```diff
@@ -5,25 +5,29 @@ on: pull_request
 jobs:
   TypeSpec-Requirement:
     name: TypeSpec Requirement
-    strategy:
-      fail-fast: false
-      matrix:
-        spec-type: [data-plane, resource-manager]
 
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v4
-        with:
-          # Required since "HEAD^" is passed to TypeSpec-Requirement.ps1
-          fetch-depth: 2
-
-      - name: Setup Node and run `npm ci`
-        uses: ./.github/actions/setup-node-npm-ci
-
-      - run: |
-          eng/scripts/TypeSpec-Requirement.ps1 `
-            -BaseCommitish HEAD^ `
-            -TargetCommitish HEAD `
-            -SpecType ${{ matrix.spec-type }}
-        shell: pwsh
+    - uses: actions/checkout@v4
+      with:
+        # Required since "HEAD^" is passed to TypeSpec-Requirement.ps1
+        fetch-depth: 2
+
+    - name: Setup Node and run `npm ci`
+      uses: ./.github/actions/setup-node-npm-ci
+
+    - run: |
+        eng/scripts/TypeSpec-Requirement.ps1 `
+          -BaseCommitish HEAD^ `
+          -TargetCommitish HEAD `
+      id: tsr-ps1
+      shell: pwsh
+
+    # Always add label artifact, even if "brownfield=false", to ensure label is removed when necessary
+    - uses: ./.github/actions/add-label-artifact
```

Comment:
For my information do you know why you need `.` in front? I would expect if it is from the repo root it would just start with `/`

Reply:
I followed the syntax in the example here:

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-an-action-in-the-same-repository-as-the-workflow

I suspect both `./` and `/` would work.  I think `./` is technically more correct, since `./` represents the current directory (assumed to be the repo root), while on Linux `/` represents the global root of the entire file system, not the repo root.